guides/source/security.md

```diff
@@ -115,9 +119,9 @@ That means the security of this storage depends on this secret (and on the diges
     production:
       secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
 
-Older versions of Rails use CookieStore, which uses `secret_token` instead of `secret_key_base` that is used by EncryptedCookieStore. Read the upgrade documentation for more information.
+Use `rails secret` command to get strong random secrets you can use. Use different secrets for different environments.
 
-If you have received an application where the secret was exposed (e.g. an application whose source was shared), strongly consider changing the secret.
+If an application's secret may have been exposed, strongly consider changing the secret. Changing the secret will expire currently active sessions.
```

Comment:
I think it might be worth keeping the "source was shared" example here: beyond the direct utility of pointing out a common way for the secret to be compromised, it provides some context on the *sorts* of things that will expose the secret -- and by implication, things that probably won't.